README.md

```diff
@@ -1,160 +1,63 @@
 [![Build Status](https://travis-ci.org/tesseract-ocr/tesseract.svg?branch=master)](https://travis-ci.org/tesseract-ocr/tesseract)
 [![Build status](https://ci.appveyor.com/api/projects/status/miah0ikfsf0j3819?svg=true)](https://ci.appveyor.com/project/zdenop/tesseract/)
 
-Note that this is possibly out-of-date version of the wiki ReadMe,
-which is located at:
 
-  https://github.com/tesseract-ocr/tesseract/blob/master/README.md
```

Comment:
README.md is part of official packages. IMO it should point to the latest online version, where could be found new information (solution for problem in release package)